File: api/core/src/main/kotlin/domain/Naslov.kt
package domain

import base.Encrypted
import base.Id
import kotlinx.serialization.Contextual
import kotlinx.serialization.Serializable

@Serializable
data class Naslov(
    override var _id: Id<Naslov> = Id(),
    var oseba_id: Id<Oseba>,
    @Contextual val drzava: Encrypted,
    @Contextual val mesto: Encrypted,
    @Contextual val zip: Encrypted,
    @Contextual val ulica: Encrypted,
    @Contextual val dodatno: Encrypted
) : Entiteta<Naslov>

File: app/src/app/services/api/openapi/models/naslov.ts
/* tslint:disable */
/* eslint-disable */
import { BsonBinary } from './bson-binary';
import { ObjectId } from './object-id';
export interface Naslov {
  '_id'?: ObjectId;
  dodatno?: BsonBinary;
  drzava?: BsonBinary;
  mesto?: BsonBinary;
  oseba_id?: ObjectId;
  ulica?: BsonBinary;
  zip?: BsonBinary;
}
